providers/fab/src/airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -301,8 +311,12 @@ class FabAirflowSecurityManagerOverride(AirflowSecurityManagerV2):
         (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_VARIABLE),
         (permissions.ACTION_CAN_DELETE, permissions.RESOURCE_VARIABLE),
         (permissions.ACTION_CAN_DELETE, permissions.RESOURCE_XCOM),
-        (permissions.ACTION_CAN_DELETE, RESOURCE_ASSET),
         (permissions.ACTION_CAN_CREATE, RESOURCE_ASSET),
+        (permissions.ACTION_CAN_DELETE, RESOURCE_ASSET),
+        (permissions.ACTION_CAN_CREATE, RESOURCE_BACKFILL),
+        (permissions.ACTION_CAN_READ, RESOURCE_BACKFILL),
```

Comment:
nit: no need, it is already part of `VIEWER`

Reply:
removed